packages/coinjoin/src/client/coordinator.ts

```diff
@@ -111,12 +110,18 @@ export const outputRegistration = (
         options,
     );
 
-export const readyToSign = (roundId: string, aliceId: string, options: RequestOptions) =>
+export const readyToSign = (
+    roundId: string,
+    aliceId: string,
+    affiliationFlag: AffiliationFlag | undefined,
+    options: RequestOptions,
+) =>
     request(
         'ready-to-sign',
         {
             roundId,
             aliceId,
+            affiliationFlag,
```

Comment:
affiliationFlag: affiliationFlag ? 'trezor' : undefined,

Reply:
move trezor to constants

Reply:
[e5cf3d2](https://github.com/trezor/trezor-suite/pull/7405/commits/e5cf3d27f5b6e5419b010108e522c09eef94a15a)